x-pack/plugins/security/public/management/users/users_grid/users_grid_page.tsx

```diff
@@ -260,6 +260,11 @@ export class UsersGridPage extends Component<Props, State> {
             {
               <EuiInMemoryTable
                 itemId="username"
+                tableCaption={i18n.translate('xpack.security.management.users.tableCaption', {
+                  defaultMessage: 'List of {count} users.',
+                  values: { count: this.state.visibleUsers.length },
+                })}
```

Comment:
A few nits on this:
- Calling a table a list could cause some confusion (especially when assistive tech will usually also then identify this as a table)
- Tables have a lot of really advanced controls for many assistive tech, so restating the `count` the more than likely redundant

So, all that said, I think we can keep this pretty simple:

```suggestion
                tableCaption={i18n.translate('xpack.security.management.users.tableCaption', {
                  defaultMessage: 'Users',
                })}
```

Reply:
I like your suggestion, thanks! I used the wording described in the original issue (#43018), but I trust you if you say we can safely simplify this.